### PR TITLE
feat(logging): Adding logrus for the logging in pod-delete experiment

### PR DESCRIPTION
**Pull request by ispeakc0de**
- Adding logrus for the logging in pod-delete experiment

- fixes: #3

### CODE REVIEW

**Comment by ispeakc0de** [comment]
Screenshot from 2020-05-19 14-29-47

File: chaoslib/litmus/pod_delete/pod-delete.go [hunk 1]
```diff
@@ -25,7 +26,7 @@ func PodDeleteChaos(experimentsDetails *types.ExperimentDetails, clients environ
 		if err != nil {
 			return err
 		}
-		klog.V(0).Infof("Killing %v pods", targetPodList)
+		log.WithFields(log.Fields{}).Infof("Killing %v pods", targetPodList)
```
Comment (ksatchit) [inline chaoslib/litmus/pod_delete/pod-delete.go:29]: can we create an abstraction for this

File: experiments/generic/pod-delete/pod-delete.go [hunk 2]
```diff
@@ -20,39 +28,42 @@ func main() {
 	
 	//Getting kubeConfig and Generate ClientSets
 	if err := clients.GenerateClientSetFromKubeConfig(); err != nil {
-		klog.V(0).Infof("Unable to Get the kubeconfig due to %v\n", err)
+		log.WithFields(log.Fields{}).Fatal("Unable to Get the kubeconfig due to %v", err)
```
Comment (rahulchheda) [inline experiments/generic/pod-delete/pod-delete.go:31]: Can we create a local or exported variable for `log.Fields{}`, and can we fill that up with some `key=value` for different logs like, `[Status]`, `[Pre-req]`

**Comment by ispeakc0de** [comment]
**Logs when TTL is disabled**
```
time="2020-05-19T09:30:01Z" level=info msg="[PreReq]: Getting the ENV for the  experiment"
time="2020-05-19T09:30:01Z" level=info msg="[PreReq]: Updating the chaos result of pod-delete experiment (SOT)"
time="2020-05-19T09:30:01Z" level=info msg="The application informations are as follows:" Label="run=nginx" Namespace=test Ramp Time=10
time="2020-05-19T09:30:01Z" level=info msg="[Status]: Verify that the AUT (Application Under Test) is running (pre-chaos)"
time="2020-05-19T09:30:01Z" level=info msg="[Status]: Checking whether application pods are in running state"
time="2020-05-19T09:30:01Z" level=info msg="nginx-7989b5f7f4-jttw2 Pod is in Running State"
time="2020-05-19T09:30:01Z" level=info msg="[Status]: Checking whether application containers are in running state"
time="2020-05-19T09:30:01Z" level=info msg=" nginx container of pod nginx-7989b5f7f4-jttw2 is in Running State"
time="2020-05-19T09:30:06Z" level=info msg="[Ramp]: Waiting for the 10s ramp time before injecting chaos"
time="2020-05-19T09:30:16Z" level=info msg="[Info]: Killing the following pods" PodList="[nginx-7989b5f7f4-jttw2]"
time="2020-05-19T09:30:21Z" level=info msg="[Wait]: Wait for the chaos interval 10s"
time="2020-05-19T09:30:31Z" level=info msg="[Status]: Verification for the recreation of application pod"
time="2020-05-19T09:30:31Z" level=info msg="[Status]: Checking whether application pods are in running state"
time="2020-05-19T09:30:31Z" level=info msg="nginx-7989b5f7f4-mg2pl Pod is in Running State"
time="2020-05-19T09:30:31Z" level=info msg="[Status]: Checking whether application containers are in running state"
time="2020-05-19T09:30:31Z" level=info msg=" nginx container of pod nginx-7989b5f7f4-mg2pl is in Running State"
time="2020-05-19T09:30:32Z" level=info msg="[Info]: Killing the following pods" PodList="[nginx-7989b5f7f4-mg2pl]"
time="2020-05-19T09:30:37Z" level=info msg="[Wait]: Wait for the chaos interval 10s"
time="2020-05-19T09:30:47Z" level=info msg="[Status]: Verification for the recreation of application pod"
time="2020-05-19T09:30:47Z" level=info msg="[Status]: Checking whether application pods are in running state"
time="2020-05-19T09:30:47Z" level=info msg="nginx-7989b5f7f4-lsslc Pod is in Running State"
time="2020-05-19T09:30:47Z" level=info msg="[Status]: Checking whether application containers are in running state"
time="2020-05-19T09:30:47Z" level=info msg=" nginx container of pod nginx-7989b5f7f4-lsslc is in Running State"
time="2020-05-19T09:30:47Z" level=info msg="[Info]: Killing the following pods" PodList="[nginx-7989b5f7f4-lsslc]"
time="2020-05-19T09:30:52Z" level=info msg="[Completion]: pod-delete chaos is done"
time="2020-05-19T09:30:52Z" level=info msg="[Ramp]: Waiting for the 10s ramp time after injecting chaos"
time="2020-05-19T09:31:02Z" level=info msg="[Confirmation]: The application pod has been deleted successfully"
time="2020-05-19T09:31:02Z" level=info msg="[Status]: Verify that the AUT (Application Under Test) is running (post-chaos)"
time="2020-05-19T09:31:02Z" level=info msg="[Status]: Checking whether application pods are in running state"
time="2020-05-19T09:31:02Z" level=info msg="nginx-7989b5f7f4-nvps2 Pod is in Running State"
time="2020-05-19T09:31:02Z" level=info msg="[Status]: Checking whether application containers are in running state"
time="2020-05-19T09:31:02Z" level=info msg=" nginx container of pod nginx-7989b5f7f4-nvps2 is in Running State"
time="2020-05-19T09:31:07Z" level=info msg="[The End]: Updating the chaos result of pod-delete experiment (EOT)"

```

**Pull request merged**